card/mod/basic_formats/set/all/all_css.rb

```diff
@@ -22,12 +22,14 @@ def show view, args
   end
 
   view :import do |_args|
-    %{\n@import url("#{_render_url items: { view: :import }}");\n}
+    voo.items[:view] = :import
+    %{\n@import url("#{_render_url}");\n}
```

Comment:
this change is fine, but I'm surprised that the previous one didn't work with your changes to the url view.